code/index.html

```diff
@@ -1,22 +1,168 @@
+              </a>
+              <a href="#" class="header__social__item">
+                <svg class="header__social__item__icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path d="M400 32H48C21.5 32 0 53.5 0 80v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V80c0-26.5-21.5-48-48-48zm-48.9 158.8c.2 2.8.2 5.7.2 8.5 0 86.7-66 186.6-186.6 186.6-37.2 0-71.7-10.8-100.7-29.4 5.3.6 10.4.8 15.8.8 30.7 0 58.9-10.4 81.4-28-28.8-.6-53-19.5-61.3-45.5 10.1 1.5 19.2 1.5 29.6-1.2-30-6.1-52.5-32.5-52.5-64.4v-.8c8.7 4.9 18.9 7.9 29.6 8.3a65.447 65.447 0 0 1-29.2-54.6c0-12.2 3.2-23.4 8.9-33.1 32.3 39.8 80.8 65.8 135.2 68.6-9.3-44.5 24-80.6 64-80.6 18.9 0 35.9 7.9 47.9 20.7 14.8-2.8 29-8.3 41.6-15.8-4.9 15.2-15.2 28-28.8 36.1 13.2-1.4 26-5.1 37.8-10.2-8.9 13.1-20.1 24.7-32.9 34z"/></svg>
+              </a>
+              <a href="#" class="header__social__item">
+                <svg class="header__social__item__icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path d="M186.8 202.1l95.2 54.1-95.2 54.1V202.1zM448 80v352c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V80c0-26.5 21.5-48 48-48h352c26.5 0 48 21.5 48 48zm-42 176.3s0-59.6-7.6-88.2c-4.2-15.8-16.5-28.2-32.2-32.4C337.9 128 224 128 224 128s-113.9 0-142.2 7.7c-15.7 4.2-28 16.6-32.2 32.4-7.6 28.5-7.6 88.2-7.6 88.2s0 59.6 7.6 88.2c4.2 15.8 16.5 27.7 32.2 31.9C110.1 384 224 384 224 384s113.9 0 142.2-7.7c15.7-4.2 28-16.1 32.2-31.9 7.6-28.5 7.6-88.1 7.6-88.1z"/></svg>
+              </a>
+            </div>
+            
+            <a class="header__title" href="#" target="_blank">Stockholm Times</a> 
+            <div class="header__actions">
+              <a class="header__actions__item" href="#" target="_blank" >SUBSCRIBE</a>
+              <a class="header__actions__item" href="#" target="_blank">LOG IN</a>
+            </div>
+    
+          </div>
+          <nav class="header__nav">
+          <!-- <nav class="header__nav">Find link for HOME and maybe add a logo ST -->
+            <a class="header__nav__item" href="#" target="_blank">NEWS</a>
+            <a class="header__nav__item" href="#" target="_blank">TECH</a>
+            <a class="header__nav__item" href="#" target="_blank">CULTURE</a>
+            <a class="header__nav__item" href="#" target="_blank">SOCIETY</a>
+            <a class="header__nav__item" href="#" target="_blank">SPORTS</a>
+            <a class="header__nav__item" href="#" target="_blank">BUSINESS</a>
+          </nav>
+    
+        </header>
+    
+        <!--FEATURE ARTICLE SECTION-->
+        <div class="Topcontent">
+    
+          <div class="Featured">
+            <h2 class="Featured__label">Featured Article</h2>
+            <div class="Featured__article">
+              <a class="Featured__image" href="#">
+                <img src="./assets/Crayfish.jpg" target="-blank" alt="Loading photo" height="400px">
```

Comment:
You even thought about the file structure, great job! Having your images in a separate folder is good practise and makes it easier for yourself!